supervisord.conf

```diff
@@ -0,0 +1,16 @@
+[supervisord]
```

Comment:
Are cron logs (by your script) also logged as part of stdout? --> I want to see these as part of `docker logs [container_name]`